app/views/notes/create.js.erb

```diff
@@ -0,0 +1,5 @@
+$('#note-form').remove();
+$('#new-note a').show();
+$('#note-list').empty();
```

Comment:
inspect what happens when you do this in the DOM. You have empty DIVs that remain